src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pyttsx3.py

```diff
@@ -0,0 +1,33 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+# pyttsx3 conditionally imports drivers module based on specific platform.
+# https://github.com/nateshmbhat/pyttsx3/blob/5a19376a94fdef6bfaef8795539e755b1f363fbf/pyttsx3/driver.py#L40-L50
+
+import sys
+
+from PyInstaller.utils.hooks import collect_submodules
```

Comment:
This unused import would make linter very unhappy.

Reply:
Grrr